shared-components/statviz/hooks/useShareableLink.ts

```diff
@@ -0,0 +1,112 @@
+import { useCallback, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { graphql } from "../../../graphql/graphql";
+import { useAuthorization } from "../../../front/src/hooks/useAuthorization";
```

Comment:
add at least a todo to clean this up:
- remove relative paths?
- move useAuthorization / useNotification to shared-components